static/src/stylesheets/base/_faux-block-link.scss

```diff
@@ -54,6 +54,12 @@ a.u-faux-block-link__overlay {
     // this line is needed as all elements have a solid black
     // background in high contrast mode
     opacity: 0;
+
+    // Override the default user agent stylesheet so we don't incur a "Layout"
+    // each time the user taps a link.
+    &:focus {
+        outline: none;
+    }
```

Comment:
The UAS adds a focus, but we never see it because the link has `opacity: 0`. It was causing a Layout which in turn triggered an expensive Update Layer Tree (~30ms)